src/main/java/de/retest/recheck/printer/AttributeDifferencePrinter.java

```diff
@@ -14,11 +15,18 @@
 
 	private static final String KEY_EXPECTED_ACTUAL_FORMAT = "%s: expected=\"%s\", actual=\"%s\"";
 
+	private static final String WARNING_FILENAME_LINE_FORMAT = ", warning=\"%s:%d\"";
```

Comment:
Why is there a distinction between warning and warnings? What does warning mean for a user? Maybe there is a more expressive name (at least for displaying it).